Makefile

```diff
@@ -11,17 +14,21 @@ up:
 vet:
 	bin/vet
 
-lint:
-	bin/lint
+lint: tools
+	$(QUARKS_UTILS)/bin/lint
 
+.PHONY: tools
```

Comment:
In contrast to most of the other targets in this Makefile, I think this one not phony, meaning it should only run if the dir is missing.

Reply:
I've made it phony because if you don't run it again, you loose the check if you are up-to-date with quarks-utils (which is what `bin/tools` does). 

The bin/tools will sync the quarks-utils dependency even if you have already checked it out.

I've give it a thought at it, and the issue would be that if you change the go.mod to point to a different version of quarks-utils,  you might have a false-sign that you are up-to-sync, while a manual intervention is required

Reply:
I read the bin/tools script too late, sorry :)

I'll approve this PR so we can move to the new scripts. But I have huge concerns that this is too much magic, not due this PR, but for the story.

Reply:
No worries, yeah I agree.. this approach ended up being very much magical, and I already start to feel that will hit us later _iff_ the projects will start to diverge...